src/validate.rs

```diff
@@ -92,9 +112,8 @@ fn validate_config(opts: &Opts, fmt: &mut Formatter) -> Option<Config> {
     };
 
     // Load
-    let paths_list: Vec<_> = paths.iter().map(|(path, _)| path).collect();
```

Comment:
This is used so to only print the paths for beauty/clarity sake. So we can use something like this
```
let paths_list: Vec<_> = paths.iter().map(<&PathBuf>::from).collect();
```